Documentation/tutorials/saving-and-resuming-tracks.md

```diff
@@ -0,0 +1,58 @@
+
+
+The file structure is as follows:
+
+```
+trial name
+├── radiographs
+|   └── ...
+├── volumes
+|   └── ...
+├── calibration
+|   └── ...
+├── xParamters
+|   └── control_settings.vie
+├── Tracking
+|   ├── {trial name}_bone1.tra
+|   ├── {trial name}_bone2.tra
+|   ├── ...
+|   └── {trial name}_bonek.tra
+└── trial name.cfg
+```
+
+### Saving a Trial Configuration
+
+To save a trial configuration, click the `File` menu and select `Save a Trial` or press `Ctrl+Shift+S` on your keyboard. This will open a dialog box that will allow you to select a location to save the trial configuration. The trial configuration will be saved as a `.cfg` file.
+
+
+### Saving Tracks
+
+Inorder for tracks to be automatically loaded when a trial is loaded, tracks must be saved in the default configuration. For more info on the different track saving options, see the [](../user-interface.md#importexport-tracking-options) section of the User Interface page.
+
+For each bone you would like to save tracks for, select the bone in the `Volumes` panel (see the [](../user-interface.md) for more information). Once you have the desired bone selected, click the `File` menu and select `Save Tracking`, press `Ctrl+S` on your keyboard, or press the `Save Tracking` button on the [](../user-interface.md#toolbar). This will open a dialog box that will allow you to select a location to save the tracks. The tracks will be saved as a `.tra` file. Ensure that you are saving the tracks in the `Tracking` subfolder of the trial you are working on.
+
+### Saving Filter Settings
+
+Inorder for filter settings to be automatically loaded when a trial is loaded, filter settings must be saved into the default location. For more info on filters, see the [](./filters.md) tutorial. Ensure that you are saving the filter settings in the `xParameters` subfolder of the trial you are working on, and that the file is named `control_settings.vie`.
+
+```{note}
+The current version of Autoscoper only supports loading one settings file for all of the cameras. If you have different filter settings for different cameras, you will need to manually load the settings for each camera after loading the trial.
+```
+
+## Resuming a Trial
+
+If everything is saved in the correct file structure, you can resume a trial by simply loading the trial configuration file (`.cfg`) that you saved earlier. To do this, click the `File` menu and select `Load a Trial`, press `Ctrl+O` on your keyboard, or press the `Open Trial` button on the [](../user-interface.md#toolbar). This will open a dialog box that will allow you to select the trial configuration file. Once you have selected the trial configuration file, click `Open` to load the trial.
```

Comment:
```suggestion
You can resume a trial by simply loading the trial configuration file (`.cfg`) (see [](#saving-a-trial-configuration)). To do this, click the `File` menu and select `Load a Trial`, press `Ctrl+O` on your keyboard, or press the `Open Trial` button on the [](../user-interface.md#toolbar). This will open a dialog box that will allow you to select the trial configuration file. Once you have selected the trial configuration file, click `Open` to load the trial.
```